cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,70 @@
 describe('Cypress application', () => {
-  before(() => {
+  const alertMessage = {
+    clickOnButton: 'You clicked a button',
+    allertAppearedAfterTime: 'This alert appeared after 5 seconds',
+    confirmAction: 'Do you confirm action?',
+    enterYourName: 'Please enter your name'
+  };
 
+  beforeEach(() => {
+    cy.visit('/')
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
-
+    cy.get('#alertButton')
+    .click();
+    cy.on('window:alert', (alert) => {
+      expect(alert).to.equal(alertMessage.clickOnButton);
   });
+});
 
   it('should have the ability to assert scheduled allert', () => {
-
+    cy.get('#timerAlertButton')
+      .click();
+    cy.on('window:alert', (alert) => {
+      expect(alert).to.equal(alertMessage.allertAppearedAfterTime);
+    cy.intercept('allertAppearedAfterTime')
+    cy.wait('@allertAppearedAfterTime');
```

Comment:
The intercept command is not correctly implemented. The intercept command is used to stub network requests and responses, not to wait for window alerts.